renderer/pages/index.tsx

```diff
@@ -19,6 +19,8 @@ const Home = () => {
   // STATES
   const [imagePath, SetImagePath] = useState("");
   const [upscaledImagePath, setUpscaledImagePath] = useState("");
+  const [formattedImagePath, setFormattedImagePath] = useState("");
+  const [formattedUpscaledImagePath, setFormattedUpscaledImagePath] = useState("");
```

Comment:
I think instead of doing this, we could directly use formatPath on the setImagePath and setUpscaledImagePath.